### DISCUSSION TITLE
Passing props in the livewire volt component not working.

### DISCUSSION THREAD

**Discussion by RazneshPrajapati** [livewire/livewire#7692] [1. Help]
I am passing props in livewire volt component.

`<livewire:employees.profile.tab-document :activeTab="$activeTab" />`

Like this. And using state to using state() function to accept the properties.

`
use function Livewire\Volt\{state};
state('activeTab');
`

Now when i try to access the property.

```
<div>
    <div>current Value: {{ $activeTab }}</div>
</div>
```

It is giving error. Undefined variable $activeTab.

Am I doing something wrong? Can anyone help me on this?

**Comment by mc4rdle** [comment]
@RazneshPrajapati did you figure this out?

**Comment by dev-rubel** [comment]
Add key props to component.
<livewire:employees.profile.tab-document :activeTab="$activeTab" _:key=""_/>
